manim/mobject/geometry/labeled.py

```diff
@@ -153,3 +219,161 @@ def __init__(
         **kwargs,
     ) -> None:
         super().__init__(*args, **kwargs)
+
+
+class LabeledPolygram(Polygram):
+    """Constructs a polygram containing a label box at its pole of inaccessibility.
+
+    Parameters
+    ----------
+    label : str | Tex | MathTex | Text
+        Label that will be displayed on the line.
+    precision : float | optional
+        The precision used by the PolyLabel algorithm.
+    font_size : float | optional
+        Control font size for the label. This parameter is only used when `label` is of type `str`.
+    label_color: ParsableManimColor | optional
+        The color of the label's text. This parameter is only used when `label` is of type `str`.
+    label_frame : Bool | optional
+        Add a `SurroundingRectangle` frame to the label box.
+    frame_fill_color : ParsableManimColor | optional
+        Background color to fill the label box. If no value is provided, the background color of the canvas will be used.
+    frame_fill_opacity : float | optional
+        Determine the opacity of the label box by passing a value in the range [0-1], where 0 indicates complete transparency and 1 means full opacity.
```

Comment:
Same suggestions as before: don't write the types again in the docstring, and use `label_config` and `frame_config` parameters.

Don't forget to include the `vertex_groups` parameter.